Instruments/python/merging_trees.py

```diff
@@ -0,0 +1,41 @@
+import uproot
+import numpy as np
+from array import array
+import ROOT 
+
+usage = 'merging_trees.py'
```

Comment:
- remove this line
- rename file to `merge_trees.py`
- add the "standard" comment on top of the file
  ```python
  # Merge multiple TTrees.
  # This file is part of https://github.com/hh-italian-group/AnalysisTools.
  ```